rllib/core/rl_trainer/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .rl_trainer import RLTrainer
```

Comment:
Please never use `.[something]` imports. Always import from the whole path.

Reply:
ok ok.